tasks/version_powershell.ps1

```diff
@@ -0,0 +1,8 @@
+if (Test-Path 'HKLM:\SOFTWARE\Puppet Labs\Puppet') {
+  $reg = $(Get-ItemProperty -Path 'HKLM:\SOFTWARE\Puppet Labs\Puppet')
+  if (Test-Path $reg.RememberedInstallDir64) {
+    type $($reg.RememberedInstallDir64+"VERSION")
+  } elseif (Test-Path $reg.RememberedInstallDir) {
+    type $($reg.RememberedInstallDir+"VERSION")
+  }
+}
```

Comment:
Should we also try `puppet --version` in case Puppet is present not from our packages? For Bolt's use-case, apply wouldn't work without `/opt/puppetlabs/puppet/bin/ruby` or `C:/Program Files/Puppet Labs/Puppet/sys/ruby/bin/ruby`.

Reply:
I think we can include some info about how we found it as well.